test/dotnet/ExpectedOutput/bcpOptions.out

```diff
@@ -366,3 +350,82 @@ bcp -w#!#in#!#bcp_source#!#destinationTable
 #!#2
 #Q#drop table sourceTable
 #Q#drop table destinationTable
+#Q#Create table sourceTable(a int, b int)
+#Q#Create table destinationTable(a int, b int default(4))
+#Q#Insert into sourceTable values (1, 1);
+#Q#Insert into sourceTable values (1, NULL);
+#Q#Insert into sourceTable values (NULL, 1);
+#Q#Insert into sourceTable values (NULL, NULL);
+#Q#Insert into sourceTable(a) values (2);
+bcp -k#!#out#!#bcp_source#!#sourceTable
+bcp -k#!#in#!#bcp_source#!#destinationTable
+#Q#Select * from sourceTable
+#D#int#!#int
+1#!#1
+1#!#
+#!#1
+#!#
+2#!#
+#Q#Select * from destinationTable
+#D#int#!#int
+1#!#1
+1#!#
+#!#1
+#!#
+2#!#
+#Q#drop table sourceTable
+#Q#drop table destinationTable
+#Q#Create table sourceTable(a int, b int default(4))
+#Q#Create table destinationTable(a int, b int default(4))
+#Q#Insert into sourceTable values (1, 1);
+#Q#Insert into sourceTable values (1, NULL);
+#Q#Insert into sourceTable values (NULL, 1);
+#Q#Insert into sourceTable values (NULL, NULL);
+#Q#Insert into sourceTable(a) values (2);
+bcp -k#!#out#!#bcp_source#!#sourceTable
+bcp -k#!#in#!#bcp_source#!#destinationTable
+#Q#Select * from sourceTable
+#D#int#!#int
+1#!#1
+1#!#
+#!#1
+#!#
+2#!#4
+#Q#Select * from destinationTable
+#D#int#!#int
+1#!#1
+1#!#
+#!#1
+#!#
+2#!#4
+#Q#drop table sourceTable
+#Q#drop table destinationTable
+#Q#Create table sourceTable(a int, b int,c as a * b)
```

Comment:
Lets have another test case with sourceTable table with a different computed column, like:
```
sourceTable(a int, b int,c as a + b)
destinationTable(a int, b int,c as a - b)
```
This will make sure that we are not directly feeding any values fetched from client